src/modules/Root/Root.jsx

```diff
@@ -10,27 +10,36 @@ import Header from '../Layout/Header/Header';
 import Footer from '../Layout/Footer/Footer';
 import APIRespondModalContainer from '../modals/APIRespondModal/apiRespondModalContainer';
 
+import LoginPage from '../pages/LoginPage/LoginPage';
 import ApiListPage from '../pages/ApiListPage/ApiListPage';
 import NewApiPage from '../pages/NewApiPage/NewApiPage';
 import EditApiPage from '../pages/EditPage/EditApiPage';
 
+import block from '../../helpers/bem-cn';
+
+import { navClassName } from '../Layout/Nav/Nav';
+
+const nav = block(navClassName('item')()); 
```

Comment:
it's over complication. Let's not import class name builder from other components